index-test.js

```diff
@@ -748,4 +748,16 @@ describe('reactElementToJSXString(ReactElement)', () => {
   </div>
 </div>`);
   });
+
+  it('should return the functionValue value when "showFunctions" is true and functionValue is a valid string', () => {
+    expect(
+      reactElementToJSXString(<div fn={() => 'value'}/>, {showFunctions: true, functionValue: 'Replace me'})
```

Comment:
@vvo I just tried it out and ran into an issue: 
`
it('should return the functionValue as a string when "showFunctions" is true and functionValue is a function', () => {
    expect(
      reactElementToJSXString(<div fn={() => 'value'}/>, {showFunctions: true, functionValue: i => i++})
  ).toEqual('<div fn={i => i++} />');
  });
`
By the time the arrow function reaches to `.toString`, it gets  transpiled to
`function functionValue(i) {
    return i++;
  }}`

Any advice on how I should go around this? Alternatively we could allow only a string.